src/app/services/code-view/angular-code-service.ts

```diff
@@ -376,21 +377,19 @@ export class AngularCodeService extends CodeService {
     }
 
     private checkForDvSample(demoFileMetadataName: string, demosBaseUrl: string): string {
-        let hasDvSample = false;
-        this.dvSamplesPaths.forEach(p => {
+        const hasDvSample = this.dvSamplesPaths.some(p => {
             if (demoFileMetadataName.includes(p)) {
                 demoFileMetadataName = demoFileMetadataName.replace(p, "");
```

Comment:
Can this code be executed inside an if (hasDvSample) ?

Reply:
The code block contains three if blocks:

```typescript
if (demoFileMetadataName.includes(p)

if (!hasDvSample)

if (hasDvSample)
```

Could we combine it into a single if else statement?

Reply:
After a call with you, we found that there is a way to combine the if blocks and simplify the code :)